README.md

```diff
@@ -26,30 +23,130 @@ dune-release help release # for help about releasing your package
 dune-release help         # for more help
 ```
 
-[doc]: https://samoht.github.com/dune-release/
+Calling `dune-release` without any argument will start the automated release process, composed of the following steps:
+- create the distribution archive;
+- publish it online with its documentation;
+- create an opam package;
+- submit it to OCaml's opam repository.
+
```

Comment:
It might be a good place for a short paragraph explaining that if the repo contains multiple opam packages, dune-release will try to release all of them by default (ie one tarball, one gh release but several opam files added to opam-repo).

Reply:
done